src/garage/sampler/rl2_worker.py

```diff
@@ -0,0 +1,59 @@
+"""Worker class used in all Samplers."""
+import numpy as np
+
+from garage.sampler.worker import DefaultWorker
+
+
+class RL2Worker(DefaultWorker):
+    """Initialize a worker.
```

Comment:
Can you describe the class in more detail here?

Reply:
Done